main.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable  no-unused-vars */
+const openNav = () => {
+  document.getElementById('mySidenav').style.width = '250px';
+};
+
+const closeNav = () => {
+  document.getElementById('mySidenav').style.width = '0';
+};
```

Comment:
- [ ] **OPTIONAL** Following the comment above about using the `id` attribute, you could use `document.querySelector('.sidenav')` and it'll deliver the same results to you.